### PR TITLE
Fixes dead people spamming "drowning" messages

### LINKED ISSUE #8563: Pools will display drowning messages on dead people
image

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
* closes https://github.com/BeeStation/BeeStation-Hornet/issues/8563

## Why It's Good For The Game
Doesn't make sense that the body keeps "fainting" after dying.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
fix: Fixed dead bodies continuing to make the "drowning" message
/:cl:

### CODE REVIEW

**Review by EvilDragonfiend — DISMISSED** [review]
Add a comment why you check dead status in the if line. It would look like a random check.

File: code/modules/pool/components/swimming.dm [hunk 1]
```diff
@@ -130,7 +130,7 @@
 		victim.emote("cough")
```
Comment (itsmeow) [inline code/modules/pool/components/swimming.dm]: Dead people don't cough or gasp either. In fact, they shouldn't even process drown ticks. Also, do `victim.stat`, crit and unconscious shouldn't gasp/fall unconcious either.

**Review by itsmeow — DISMISSED** [review]

File: code/modules/pool/components/swimming.dm [hunk 2]
```diff
@@ -125,6 +125,8 @@
 /datum/component/swimming/proc/drown(mob/living/victim)
 	if(victim.losebreath < 1)
 		victim.losebreath += 1
+	if(victim.stat > CONSCIOUS)
```
Comment (PowerfulBacon) [inline code/modules/pool/components/swimming.dm:129]: This needs to be >= if you want unconcious or dead people to not emote. Also this removes the victim being made unconcious while unconcious when drowning which makes drowning trivial to escape.

Comment (CydiaLamiales) [reply code/modules/pool/components/swimming.dm:129]: Tried a different approach, could you take a look?

**Review by PowerfulBacon — APPROVED** [review]
Could wrap the shouldemote = FALSE if check into one line but this works and is fine

**Pull request merged**